### DISCUSSION TITLE
可否在翻译中增加音标？

### DISCUSSION THREAD

**Discussion by pengfei-lyu** [STranslate/STranslate#129] [Ideas]
感谢作者的奉献，让WIN也用到了这种小巧方便的小工具。
有个小提议，可以参考一下。在BOB中，调用三方API时，会显示出音标，这个功能可以方便陌生单词的学习，不知道作者有没有兴趣增加一下？
![image](https://github.com/user-attachments/assets/5ab77442-19ba-4ee1-a9d8-1b5ad805776e)

**Comment by ZGGSONG** [comment]
![image](https://github.com/user-attachments/assets/ba9c9667-db30-4d55-bc4e-8e4bc52dc410)
这属于词典功能，目前词典仅接入简明英汉词典 离线

**Reply by pengfei-lyu** [reply]
OK，足矣~
